help/en/releasenotes/current-draft-note.shtml

```diff
@@ -447,20 +447,7 @@
     <h3>Turnouts, Lights, Sensors and other elements</h3>
         <a id="TLae" name="TLae"></a>
         <ul>
-            <li>The handling of Turnout system names has been updated
```

Comment:
This comment must be restored since this is a post 4.14 change that needs to be in the 4.16 release notes.

Reply:
I'm not sure why that's showing up as a change.  When 4.15.5 was released, that file was rolled to a blank one and that warning only appears in the "old warnings" part at the top of the JMRI/website release note file.

If you look at http://jmri.org/help/en/releasenotes/current-draft-note.shtml  that comment isn't there.

Reply:
Ah, I think I understand:  `This branch is 2 commits ahead, 343 commits behind JMRI:master.`